package-lock.json

```diff
@@ -9,7 +9,7 @@
       "version": "1.0.1",
```

Comment:
I imagine we're going to be releasing 1.0.2 pretty soon. You might want to consider updating the version in package.json and package-lock.json to 1.0.2 as part of this PR.

Reply:
I'd planned to increment to 1.0.2 in a separate commit, but it makes sense to do it in this commit as these changes are the last meant to be included in the 1.0.2 release.